src/sidebar/store/modules/test/filters-test.js

```diff
@@ -45,6 +45,15 @@ describe('sidebar/store/modules/filters', () => {
         assert.isFalse(filterState.focusActive);
         assert.isUndefined(filterState.focusFilters.user);
       });
+
+      it('un-sets and deactivates the user focus filter when null user value is provided', () => {
+        store.toggleFocusMode(true);
+        store.changeFocusModeUser({ user: null });
```

Comment:
This test passes an object with a `user` property, but the argument to `changeFocusModeUser` is a `FocusUserConfig` which doesn't have a `user` property. I think you've passed a `FocusConfig` here instead?